purchase_order_qweb_report_adj/reports/purchase_order_templates.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document_adj"
+        inherit_id="purchase.report_purchaseorder_document"
+        priority="100"
+    >
+        <xpath expr="//div[@t-if='o.partner_ref']" position="attributes">
+            <attribute name="t-att-style">'display: none!important'</attribute>
```

Comment:
```suggestion
            <attribute name="t-att-style">'display: none'</attribute>
```
Is this necessary?

Reply:
the purpose is to add translation of the "your order reference" label, so i put the same code here and hide the original one.

Reply:
Sorry, I mean `!important` is necessary?

Reply:
i just check it again and its ok without important. thanks Kanda san